LogForwarder/BlobForwarder/function.json

```diff
@@ -1,7 +1,7 @@
 {
   "bindings": [
     {
-      "name": "myBlob",
+      "name": "logMessages",
```

Comment:
What is the reason behind this?

Reply:
Not a necessary changes. Since the code is now common, I thought of unifying the variable that collects the logs messages i.e the second argument to the index.js main function. With this changes, there won't be any error even if we access the log contents via context.bindings.logMessages.